### ISSUE TITLE
seed used in the model

### ISSUE THREAD

**Issue by gxxxzhang** [yangzhou12/NCRL#1]
When I reproduced the model, the experimental results were lower than expected. So, can I ask you the seed used in your model?

**Comment by yangzhou12** [comment]
Thanks for your interest in our work. We use the same seed (66) in ATLOP to conduct the experiments for the DocRED dataset. In our paper, there are two settings: 1. Training on the human annotated dataset from scratch; 2. Fine-tuning the checkpoint pre-trained on the distantly supervised dataset, leading to the results on the official leaderboard.

To reproduce the results for the first setting, please refer to Table 7 in the appendix of our paper for the specific parameter setup.

To reproduce the results for the second setting, you should first do a pre-training step **with our NCLR** on the distantly-supervised DocRED dataset and then fine-tune the checkpoint on the human annotated dataset. 

According to our experience, pre-training with only one epoch is enough to get reasonably good results. For fine-tuning, you can try a learning rate of 5e-6 for better performance. This value is smaller than the learning rate of 1e-5 used when training from scratch on the human annotated dataset. However, using a learning rate of 1e-5 should also yield similar results.

Here is the link of the distant pre-trained checkpoint we used for fine-tuning on the human annotated dataset:
https://drive.google.com/file/d/1sqWx_falgX_oeEHsSLoSseGuDMNvfqHq/view?usp=sharing

Please let me know if you have any further questions. Thanks.
